core/runtime/v2/task_manager.go

```diff
@@ -142,6 +144,18 @@ func (m *TaskManager) Create(ctx context.Context, taskID string, opts runtime.Cr
 	}
 
 	t, err := shimTask.Create(ctx, opts)
+	if err != nil {
+		if errdefs.IsNotImplemented(err) {
+			log.G(ctx).WithError(err).
+				WithField("id", taskID).
+				Warning("failed to call task.Create and downgrade client API version to try")
```

Comment:
nit: "failed to call task.Create, downgrading client API version to try again"

Reply:
updated